code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -37,20 +37,22 @@
 /obj/structure/bed/chair/verb/rotate()
 	set name = "Rotate Chair"
 	set category = "Object"
-	set src in oview(1)
+	set src in view(0)
 
 	if(CONFIG_GET(flag/ghost_interaction))
-		setDir(turn(src.dir, 90))
-		return
-	else
-		if(istype(usr, /mob/living/simple_animal/mouse))
-			return
-		if(!usr || !isturf(usr.loc))
-			return
-		if(usr.stat || usr.is_mob_restrained())
-			return
+		setDir(turn(dir, 90))
+		return FALSE
+	
+	var/mob/living/carbon/user = usr
+
+	if(!istype(user) || !isturf(user.loc) || user.stat || user.is_mob_restrained())
```

Comment:
Use `user.is_mob_incapacitated()` instead of `user.stat || user.is_mob_restrained()`, it covers stun and knockdown statuses in additions of handcuffs and mob stat.